util/exercise/src/cmd/templates/test_file.rs

```diff
@@ -0,0 +1,52 @@
+{% import "macros.rs" as macros -%}
+
+//! Tests for {{ exercise }}
+//!
+//! Generated by [utility][utility] using [canonical data][canonical_data]
+//!
+//! [utility]: https://github.com/exercism/rust/tree/master/util/exercise
+//! [canonical_data]: https://raw.githubusercontent.com/exercism/problem-specifications/master/exercises/{{ exercise }}/canonical-data.json
+
+{% for comment in comments -%}
+    /// {{ comment }}
+{% endfor -%}
+
+{% if use_maplit -%}
+use maplit::hashmap;
+{% endif -%}
+
+{# Prepare an array (global) to store the properties. Also, don't ignore the first case. -#}
+{% set properties = [] -%}
+{% set dont_ignore = true -%}
```

Comment:
Maybe `first_test_case` would be clearer than `dont_ignore`, as a variable name?

Reply:
Yes, yes it would. Thank you! :)